src/main/resources/bin/export_solr_registry_pds3.py

```diff
@@ -0,0 +1,210 @@
+#!/usr/bin/env python
+import logging
+import os
+import re
+
+from pandas import read_csv
+from argparse import ArgumentParser, RawDescriptionHelpFormatter
+from typing import TextIO
+from uuid import uuid4
+
+from solr_to_es.solrSource import SlowSolrDocs  # type: ignore
+from xml.sax.saxutils import escape
+
+# Constants
+SKIP_FIELDS = ['score', 'timestamp', 'search_id']
+SOLR_URL = "https://pds.nasa.gov/services/search/search"
+MAX_RETRIES = 5
+DOCS_PER_FILE = 1000
+SOLR_DOC_PREFIX = "solr_doc_deprecated"
+PACKAGE_ID = uuid4()
+
+# Globals
+_known_pds4_dict = {}
+_missing_context = {'target_ref': {}, 'instrument_ref': {}, 'instrument_host_ref': {}, 'investigation_ref': {}}
+
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+def open_doc(output_path, file_counter):
+    output_filepath = os.path.join(output_path, f"{SOLR_DOC_PREFIX}.{str(file_counter)}.xml")
+    solr_doc = open(output_filepath, "w")
+    logger.info("Writing solr doc: " + output_filepath)
+    solr_doc.write("<add>\n")
+    return solr_doc
+
+def close_doc(solr_doc):
+    if solr_doc:
+        solr_doc.write("</add>\n")
+        solr_doc.close()
+
+def run(query: str, db_path: str, output_path: str):
+    doc_counter = 0
+    file_counter = 0
+    current_solr_doc = None
+
+    # Init PDS3 database dict
+    known_database_dict = init_db(db_path)
+
+    solr_itr = SlowSolrDocs(SOLR_URL, query, rows=500)
+    for doc in solr_itr:
+        # Increment file counter, as needed
+        if doc_counter % DOCS_PER_FILE == 0:
+            close_doc(current_solr_doc)
+            file_counter += 1
+            current_solr_doc = open_doc(output_path, file_counter)
+
+        current_solr_doc.write("<doc>\n")
+        current_solr_doc.write(f"<field name='package_id'>{PACKAGE_ID}</field>\n")
```

Comment:
Writing XML "by hand" always makes me nervous; is there any chance `PACKAGE_ID` might contain `<`, `&`, `'`', `"`, etc.?